components/sensor/m5stack_8angle.rst

```diff
@@ -0,0 +1,155 @@
+Rotary Encoder Sensor
+=====================
+
+.. seo::
+    :description: Setting up the ``m5stack_8angle`` input device with 8 knobs.
+    :image: m5stack_8angle.png
+
+Component/Hub
+-------------
+
+The ``m5stack_8angle`` platform allows to use the [m5angle](https://docs.m5stack.com/en/unit/UNIT%208Angle) input device with ESPHome. 
+It has 8 knobs, a switch and can individually drive 9 RGB LEDs. 
+
+.. figure:: images/m5stack_8angle.jpg
+    :align: center
+    :width: 75.0%
+
+    The m5stack_8angle unit.
+
+The ``m5stack_8angle`` component communicates through an :ref:`I²C <i2c>` bus and uses a default address of 0x43. 
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    
+    i2c:
+      sda: 26
+      scl: 32
+      scan: false
+      id: bus_external
+      frequency: 400kHz
```

Comment:
We don't include dependency yaml in the examples. It is on the user to learn what is needed by reading the pages correctly =)
```suggestion
```